### DISCUSSION TITLE
rootcoord send tt to all channels too slowly

### DISCUSSION THREAD

**Discussion by ownlu** [milvus-io/milvus#13777] [Q&A and General discussion]
Search slowly

**Comment by ownlu** [comment]
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:33.747 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":493}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:34.301 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":551}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:35.137 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":832}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:35.668 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":528}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:36.569 +00:00","caller":"session_util.go:277","message":"SessionUtil GetSessions","prefix":"IndexNode","resp":{"header":{"cluster_id":14841639068965178418,"member_id":10276657743932975437,"revision":422403,"raft_term":5},"kvs":[{"key":"YnktZGV2L21ldGEvc2Vzc2lvbi9JbmRleE5vZGUtMjk=","create_revision":421766,"mod_revision":421766,"version":1,"value":"eyJTZXJ2ZXJJRCI6MjksIlNlcnZlck5hbWUiOiJJbmRleE5vZGUiLCJBZGRyZXNzIjoiMTcyLjE5LjAuNDoyMTEyMSJ9","lease":7587859309586971952}],"count":1}}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.658 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":987}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.893 +00:00","caller":"server.go:420","message":"Time tick lag behind for more than 1 minutes","channel":"by-dev-rootcoord-dml_3_429801518499364865v1","tt":"2021/12/20 07:20:50.874 +00:00"}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:36.963 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":302}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:37.424 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":459}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:37.547 +00:00","caller":"index_coord.go:628","message":"IndexCoord recycleUnusedIndexFiles","Need recycle tasks num":0}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:37.965 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":539}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:38.453 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":485}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:38.791 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":336}
milvus-standalone | {"level":"WARN","time":"2021/12/20 08:53:39.291 +00:00","caller":"timeticksync.go:262","message":"rootcoord send tt to all channels too slowly","chanNum":2,"span":497}
milvus-standalone | {"level":"DEBUG","time":"2021/12/20 08:53:39.586 +00:00","caller":"session_util.go:277","message":"SessionUtil GetSessions","prefix":"IndexNode","resp":{"header":{"cluster_id":14841639068965178418,"member_id":10276657743932975437,"revision":422406,"raft_term":5},"kvs":[{"key":"YnktZGV2L21ldGEvc2Vzc2lvbi9JbmRleE5vZGUtMjk=","create_revision":421766,"mod_revision":421766,"version":1,"value":"eyJTZXJ2ZXJJRCI6MjksIlNlcnZlck5hbWUiOiJJbmRleE5vZGUiLCJBZGRyZXNzIjoiMTcyLjE5LjAuNDoyMTEyMSJ9","lease":7587859309586971952}],"count":1}}

**Comment by ownlu** [comment]
The longer the service is established, the slower the search

**Comment by yhmo** [comment]
@ownlu 
You'd better create an issue for this. An issue can be assigned, can be labeled so that the QA team can trace it.
